router/pkg/config/config.schema.json

```diff
@@ -52,6 +52,24 @@
             }
           }
         },
+        "kv": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["url", "id"],
+            "additionalProperties": false,
+            "properties": {
```

Comment:
Should we add kv type to distinguish kvs? E.g. we could use Redis or NATS as the kv.

Reply:
@StarpTech 
```
storage_providers:
  kv: 
    - id: "my_redis"
      url: "redis://localhost:6379"
      type: redis
    - id: "my_nats"
      url: "nats-url"
      type: nats
```

Sound ok?

Reply:
Why do you think this necessary to specify? I think I'm lacking context here.

Reply:
@StarpTech So does that mean you don't think we need a new `kv` section in `storage_providers`? I'd argue that we should have one, but I do agree that i don't think we need to provide the `type` in the `kv`'s